hardware/Makefile

```diff
@@ -382,7 +382,7 @@ cloud_merge: allow_cloud_merge $(SNAP_BASE_DCP) $(SNAP_ACTION_DCP)
 pslse: prepare_logs
 	@echo -e "[COMPILE PSLSE ......] start `date +"%T %a %b %d %Y"`"
 	@$(SNAP_ROOT)/check_pslse.sh > $(LOGS_DIR)/compile_pslse.log 2>&1
-	@$(MAKE) -C $(PSLSE_ROOT) >> $(LOGS_DIR)/compile_pslse.log 2>&1; \
+	@$(MAKE) -C $(PSLSE_ROOT) clean all >> $(LOGS_DIR)/compile_pslse.log 2>&1; \
```

Comment:
Hm, I don't like that.
I personally would not want to rebuild PSLSE if it is already available.
I have two different PSLSE instances that I am using for P8 and for P9 from various sandboxes of which some are CAPI 1.0 while others are CAPI 2.0

Reply:
This recompilation takes 3 to 4 seconds , allows user to have only one PSLSE_ROOT variable set for all cases and even more prevents errors found after having built the model.
I don't understand why this wouldn't be an improvement!